src/org/scada_lts/config/ScadaConfig.java

```diff
@@ -137,41 +173,127 @@ public Properties getConf() {
+					croneUpdateCachePointHierarchy = Optional.of(result);
+				} else if (CRONE_UPDATE_CACHE_DATA_SOURCES_POINTS.equals(propertyName)) {
+					croneUpdateCacheDataSourcePoints = Optional.of(result);
+				} else if (ACL_SERVER.equals(propertyName)) {
+					aclServer = Optional.of(result);
+				}
+			}
+		} catch (Exception e) {
+			LOG.trace("propertyName:"+propertyName +" e:"+e.getMessage());
+		}
+		return result;
 	}
 	
 	/**
 	 * Get property value of type boolean with default value
 	 * @param propertyName
-	 * @param defaultValues
+	 * @param defaultValue
 	 * @return
 	 */
-	public Boolean getBoolean(String propertyName, boolean defaultValues) {
-		Boolean result = (Boolean) defaultValues;
+	public Boolean getBoolean(String propertyName, boolean defaultValue) {
+		Boolean result = (Boolean) defaultValue;
 		try {
-			String propertyValue = getProperty(propertyName);
-			result = (Boolean) Boolean.parseBoolean(propertyValue);
-			LOG.trace("propertyName:"+propertyName+" value:"+result);
+			if (ENABLE_CACHE.equals(propertyName) && cacheEnabled.isPresent()) {
+				return cacheEnabled.get();
+			} if (USE_CACHE_DATA_SOURCES_POINTS_WHEN_THE_SYSTEM_IS_READY.equals(propertyName) && useCacheDataSourcesPointsWhenTheSystemIsReady.isPresent()) {
+				return useCacheDataSourcesPointsWhenTheSystemIsReady.get();
+			} else if (USE_ACL.equals(propertyName) && useACL.isPresent()) {
+				return useACL.get();
+			} else if (HTTP_RETRIVER_DO_NOT_ALLOW_ENABLE_REACTIVATION.equals(propertyName) && httpRetriverDoNotAllowEnableReactivation.isPresent()) {
+				return httpRetriverDoNotAllowEnableReactivation.get();
+			} else {
+				String propertyValue = getProperty(propertyName);
+				result = (Boolean) Boolean.parseBoolean(propertyValue);
+				LOG.trace("propertyName:" + propertyName + " value:" + result);
+				if (ENABLE_CACHE.equals(propertyName)) {
+					cacheEnabled = Optional.of(result);
+				} else if (USE_CACHE_DATA_SOURCES_POINTS_WHEN_THE_SYSTEM_IS_READY.equals(propertyName)) {
+					useCacheDataSourcesPointsWhenTheSystemIsReady = Optional.of(result);
+				} else if (USE_ACL.equals(propertyName)) {
+					useACL = Optional.of(result);
+				} else if (HTTP_RETRIVER_DO_NOT_ALLOW_ENABLE_REACTIVATION.equals(propertyName)) {
+					httpRetriverDoNotAllowEnableReactivation = Optional.of(result);
+				}
+			}
 		} catch (Exception e) {
-			LOG.trace("propertyName:"+propertyName+" value:"+defaultValues);
-			result = defaultValues;
+			LOG.trace("propertyName:"+propertyName+" value:"+defaultValue+" e:"+e.getMessage());
+			result = defaultValue;
 		}
 		return result;
 	}
 	
 	/**
 	 * Get property value of type long with default value
 	 * @param propertyName
-	 * @param defaultValues
+	 * @param defaultValue
 	 * @return
 	 */
-	public Long getLong(String propertyName, int defaultValues) {
-		Long result = new Long(defaultValues);
+	public Long getLong(String propertyName, int defaultValue) {
+		Long result = new Long(defaultValue);
```

Comment:
`Long result = Long.valueOf(defaultValue); `

Reply:
Resolved in 09ca4defa6a9faac2f993f0143191dc33680c677